File: src/dataset/python/word-count/claude/ClaudeSonnet4_word-count_v3.py
def count_words(sentence):
    import re
    from collections import Counter
    
    normalized = re.sub(r'[^\w\s\']', ' ', sentence.replace('\n', ' ').replace('\t', ' '))
    words = [word.strip("'") for word in normalized.lower().split() if word.strip("'")]
    
    return dict(Counter(words))

File: src/dataset/javascript/word-count_exercism-javascript-programmiri/claude/ClaudeSonnet4_word-count_v3.js
export class Words {
  count(string) {
    if (!string) return {};
    
    const words = string.toLowerCase().trim().split(/\s+/);
    const wordCount = {};
    
    for (const word of words) {
      if (word) {
        wordCount[word] = (wordCount[word] || 0) + 1;
      }
    }
    
    return wordCount;
  }
}